charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -39,3 +69,48 @@ groups:
 
   - record: seed:container_network_transmit_bytes_total:sum_cp
     expr: sum(seed:container_network_transmit_bytes_total:sum_by_pod{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for the sum of all control-plane requests
+  - record: seed:kube_pod_container_resource_requests_cpu_cores:sum_cp
+    expr: sum(seed:kube_pod_container_resource_requests_cpu_cores:sum_by_pod{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  - record: seed:kube_pod_container_resource_requests_memory_bytes:sum_cp
+    expr: sum(seed:kube_pod_container_resource_requests_memory_bytes:sum_by_pod{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for the sum of the entire seed vpa recommendations
+  - record: seed:vpa_status_recommendation:sum
+    expr: sum(seed:vpa_status_recommendation:sum_by_target{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for the sum of the entire seed hvpa applied recommendations
+  - record: seed:hvpa_status_applied_vpa_recommendation:sum
+    expr: sum(seed:hvpa_status_applied_vpa_recommendation:sum_by_target{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for node usage
+  - record: seed:node_cpu_seconds_total:sum_by_node
+    expr: sum(rate(node_cpu_seconds_total{mode!="idle"}[5m])) by (node)
```

Comment:
`node_cpu_seconds_total` comes from the node-exporter. Neither this prometheus nor the aggregate prometheus are configured to scrape the node-exporter. Same thing for below metrics

Reply:
This data is available in the shoot prometheus (of the seed in the soil). I was thinking if it could somehow be federated here. If not then I will remove this.